### PR TITLE
feat: [#997] transfomer improvements

### LINKED ISSUE #997: Transformer improvements
1) Fix Dockerfile - update java 
2) include alternativeIdentifiers in data
3) Celery task status should be influenced also by request status sent to Solr

### PR DESCRIPTION
**Pull request by Michal-Kolomanski**
closes #997 

1) Alternative identifiers handled for trainings & guidelines - they are in the data
2) Raw input schemas checked before transformation! If something changes in the input data we will know. Moreover, if that will be a new field - it won't influence the final output.
3) output data - take only those columns which are present in an expected output data schema
4) The status of sending to solr output data influences the status of the whole celery task (when sending data to solr fails, the celery task will also fail - before it wasn't the case)

### CODE REVIEW

**Review by wiktorflorian — COMMENTED** [review]
Looks really good for me, nice job! However, I have a small problem with the Solr schema because you use a copy field, and the source is not in the schema. Also, out of curiosity, I have a question about the convention. More information in specific changes comments.

File: solr/config/configsets/all_collection_oag56_v203/managed-schema
[the remark this answers is not in the document]
Comment (Michal-Kolomanski) [reply solr/config/configsets/all_collection_oag56_v203/managed-schema:1]: Thanks! Fixed.

File: transform/app/transform/transformers/training.py
[the remark this answers is not in the document]
Comment (Michal-Kolomanski) [reply transform/app/transform/transformers/training.py:1]: `col` is also a function name from pyspark. Thats way I wanted to be very explicit that is not that function

[the remark this answers is not in the document]
Comment (wiktorflorian) [reply transform/app/transform/transformers/training.py:1]: Thanks! Good to know :)

**Review by wiktorflorian — APPROVED** [review]

**Pull request merged**